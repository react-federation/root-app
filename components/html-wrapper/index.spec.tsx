import { BrowserRouter } from 'react-router-dom'
import setup from 'utils/test/setup'
import HtmlWrapper from '.'

describe('HtmlWrapper', () => {
  it('should match snapshot', () => {
    const { asFragment } = setup(HtmlWrapper, {}, BrowserRouter)

    expect(asFragment()).toMatchSnapshot()
  })
})
