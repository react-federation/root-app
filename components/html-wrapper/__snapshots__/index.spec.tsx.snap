// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HtmlWrapper should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  margin-top: 1rem;
  padding-top: 0.5rem;
}

.emotion-0 a {
  font-weight: 600;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<section
    class="emotion-0"
  />
</DocumentFragment>
`;
